scripts/envDFL.sh

```diff
@@ -0,0 +1,243 @@
+#!/bin/bash
+#
+# Copyright (c) 2021, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+
+# This file aimes to encapsulate processing for users to compiler and use DFL
+
+HERE=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+
+error_exit() {
+    echo "${1:-"Unknown Error"}" 1>&2
+    exit 1
+}
+
+export_var_env() {
+    local var="$@"
+    local name=${var%%=*}
+    local value="${var#*=}"
+
+    if ! `expr $name : "DYNAFLOW_LAUNCHER_.*" > /dev/null`; then
+      error_exit "You must export variables with DYNAWO prefix for $name."
+    fi
+
+    if eval "[ \"\$$name\" ]"; then
+      eval "value=\${$name}"
+      ##echo "Environment variable for $name already set : $value"
+      return
+    fi
+
+    if [ "$value" = UNDEFINED ]; then
+      error_exit "You must define the value of $name"
+    fi
+    export $name="$value"
+}
+
+ld_library_path_remove() {
+    export LD_LIBRARY_PATH=`echo -n $LD_LIBRARY_PATH | awk -v RS=: -v ORS=: '$0 != "'$1'"' | sed 's/:$//'`;
+}
+
+define_options() {
+    export_var_env DYNAFLOW_LAUNCHER_USAGE="Usage: `basename $0` [option] -- program to deal with DynaFlow Launcher
+
+where [option] can be:
+        =========== Build
+        build-user                              build DynaFlow Launcher
+        build-tests-coverage                    build DynaFlow Launcher and run coverage
+
+        =========== Tests
+        tests                                   launch DynaFlow Launcher unit tests
+
+        =========== Launch
+        launch-dir [dir]                        launch DynaFlow Launcher with:
+                                                - network filepath = <dir>/network.xml (only IIDM is supported)
+                                                - configuration filepath = <dir>/config.json
+        launch [network] [config]               launch DynaFlow Launcher
+
+        =========== Others
+        help                                    show all available options
+        version                                 show DynaFlow Launcher version
+        reset-environment                       reset all environment variables set by DynaFlow Launcher
+"
+}
+
+help() {
+    define_options
+    echo "$DYNAFLOW_LAUNCHER_USAGE"
+}
+
+set_commit_hook() {
+    $HERE/set_commit_hook.sh
+}
+
+set_environment() {
+    # dynawo vars
+    export DYNAWO_INSTALL_DIR=$DYNAWO_HOME
```

Comment:
Please always use export_var_env or export_var_env_force

Reply:
I cannot use a version of export_var_env_force here because I set DYNAWO variables, not DYNAFLOW_LAUNCHER variables